### DISCUSSION TITLE
[ASK] Can i colorize sub-fields after did parseJson to parent field

### DISCUSSION THREAD

**Discussion by bekbasov** [stern/stern#188] [Q&A]
ok let's look at your README's example
```
stern . --template='{{.PodName}}/{{.ContainerName}} {{with $d := .Message | parseJSON}}[{{$d.level}}] {{$d.message}}{{end}}{{"\n"}}' backend
```
can i colorize to .ContainerColor one/several of sub-fields? for example: `$d.message`
can you provide code for this please?

**Comment by superbrothers** [comment]
Are the following what you expected?

<img width="1121" alt="image" src="https://user-images.githubusercontent.com/230185/190890390-04fd6975-83b0-43b6-94e9-889cfd1599ff.png">

```
stern --template='{{.PodName}}/{{.ContainerName}} {{with $d := .Message | parseJSON}}{{color $.ContainerColor $d.message}}{{end}}{{"\n"}}' .
```
